<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NeXus Constructor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>NeXus File Layout</string>
         </attribute>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>502</width>
            <height>481</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Add component</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Components</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QQuickWidget" name="quickWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="resizeMode">
         <enum>QQuickWidget::SizeRootObjectToView</enum>
        </property>
        <property name="source">
         <url>
          <string>file:./view.qml</string>
         </url>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_NeXus_file"/>
    <addaction name="actionExport_to_NeXus_file"/>
    <addaction name="actionExport_to_Filewriter_JSON"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Filewriter_JSON">
   <property name="text">
    <string>Show Filewriter JSON</string>
   </property>
  </action>
  <action name="actionHide_JSON_Pane">
   <property name="text">
    <string>Hide JSON Pane</string>
   </property>
  </action>
  <action name="actionOpen_NeXus_file">
   <property name="text">
    <string>Open NeXus file</string>
   </property>
  </action>
  <action name="actionExport_to_NeXus_file">
   <property name="text">
    <string>Export to NeXus file</string>
   </property>
  </action>
  <action name="actionExport_to_Filewriter_JSON">
   <property name="text">
    <string>Export to Filewriter JSON</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header>QtQuickWidgets/QQuickWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
